README.md

```diff
@@ -1 +1,65 @@
-# collectd-couchbase
\ No newline at end of file
+# Collectd Couchbase Plugin
+
+## Introduction
+
+collectd-couchbase is a [collectd](http://www.collectd.org/) plugin that
+collects statistics from Couchbase.
+
+
+## Requirements
+
+* Collectd 4.9 or later (for the Python plugin)
+* Python 2.7 or later
+* Couchbase 4.1 or later
+
+## Configuration
+
+The following mandatory configuration options describe how the plugin will
+connect to the Couchbase Server:
+
+* CollectTarget - Required option. Define the plugin running instance, has two options: 'NODE' - 
+get nodes related metrics or 'BUCKET' - get bucket related metrics. Target 'BUCKET' requires to have username and
+password to connect to selected bucket
+* Host - Required option. Hostname or IP address of the Couchbase server, default is 'localhost'
+* Port - Required option. The port of the Couchbase server, default is '8091'
+
+The following boolean configuration options may be added to enable collection
```

Comment:
This line needs to be updated since these aren't boolean options. Maybe something like 

"The following optional settings may be specified to control the behavior of the plugin"